clang/include/clang/Format/Format.h

```diff
@@ -922,8 +922,23 @@ struct FormatStyle {
     ///   };
     ///   int f();
     ///   int f() { return 1; }
+    ///   int foooooooooooooooooooooooooooooooooooooooooooooooo::
```

Comment:
I'd go with short names, and just name them like `LongName` or similar and just break. This is documentation which should convey the settings, and must not break with LLVMs standard column limit.